apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -0,0 +1,301 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import pluralize from 'pluralize'
+import styled from 'styled-components'
+import { getElectionLocales } from '@votingworks/ballot-encoder'
+
+import { DEFAULT_LOCALE } from '../config/globals'
+import {
+  getBallotPath,
+  getPrecinctById,
+  getHumanBallotLanguageFormat,
+} from '../utils/election'
+
+import AppContext from '../contexts/AppContext'
+import HandMarkedPaperBallot from './HandMarkedPaperBallot'
+import Modal from './Modal'
+import Button from './Button'
+import Prose from './Prose'
+import LinkButton from './LinkButton'
+import Loading from './Loading'
+import { Monospace } from './Text'
+import { UsbDriveStatus } from '../lib/usbstick'
+import USBControllerButton from './USBControllerButton'
+
+import * as workflow from '../workflows/ExportElectionBallotPackageWorkflow'
+import { generateFilenameForBallotExportPackage } from '../utils/filenames'
+
+const USBImage = styled.img`
+  margin-right: auto;
```

Comment:
I had to add these in order to center the image

Reply:
Tried to get this working in a centered text tag and couldn't figure it so just leaving it, we'll need the styled component for the height anyway